packages/api-electron/src/preload/shims.ts

```diff
@@ -1,13 +1,23 @@
 const remote = require('electron').remote;
 const eNotify = remote.require('electron-notify');
+const PERMISSION_GRANTED = "granted";
```

Comment:
So `notification.ts` I guess.

Should we still be "shimming" the browser API like this then? For the other APIs we've exported a class under the `ssf` namespace.

i.e. instead of shimming `window.Notification`, should we just export `ssf.Notification` as a class, and implement that in all three containers?